internal/store/testing_store.go

```diff
@@ -63,3 +63,22 @@ func WaitForAction(t testing.TB, typ reflect.Type, getActions func() []Action) A
 		getActions())
 	return nil
 }
+
+// for use by tests (with a real channel-based store, NOT a TestingStore). Assert that
+// we don't see an action of the given type
+func AssertNoActionOfType(t testing.TB, typ reflect.Type, getActions func() []Action) Action {
+	start := time.Now()
+	timeout := 150 * time.Millisecond
+
+	for time.Since(start) < timeout {
+		actions := getActions()
+		for _, a := range actions {
+			if reflect.TypeOf(a) == typ {
+				t.Fatalf("Found action of type %s where none was expected: %+v", typ.Name(), a)
+			} else if la, ok := a.(LogAction); ok {
+				fmt.Println(string(la.Message()))
```

Comment:
Is this supposed to be here?

Reply:
I believe so -- compare to [WaitForAction](https://github.com/windmilleng/tilt/blob/0cbfda50385c2c21f9a0fb0d1eb5e751d6b7e533/internal/store/testing_store.go#L57). I DID  just copy that function over so I'm not positive if this bit is necessary/necessary for my test case, but if memory serves, we need it to see what we're logging when we're not running a real reducer, cuz otherwise those log actions don't get processed.